butterknife-compiler/src/main/java/butterknife/compiler/ButterKnifeExtension.java

```diff
@@ -0,0 +1,9 @@
+package butterknife.compiler;
```

Comment:
I think this should be refactored to the `butterknife` or `butterknife-annotations` artifact so that it's on the classpath while writing an app?